### ISSUE TITLE
Clean Up From SUW

### ISSUE THREAD

**Issue by braidn** [treymiller/eMG-frontend#1]
Comments created to issue from 455ddb6
- [x] delete text from the MailChip snippet code that says "Sign up to be notified when we launch the Beta”
- [x] turn off the My Account link in the header, since it’s broken right now
- [ ] take out some of the black horizontal space at the footer around the Twitter link, and change text to “You shouldn’t follow us on…” and ideally put the Twitter logo/link on the same horizontal line

**Comment by braidn** [comment]
@treymiller I need to move to a more sensible asset pipeline, or really tweak what we have going on here. It is too hard to test locally. Assets are basically images, css, js, etc. I will work on this throughout the day.

**Comment by treymiller** [comment]
Awesome, thanks.

Trey
Triad Tech Services
336.912.2110
triadts.com

On Apr 7, 2014, at 9:40 AM, Braden Douglass notifications@github.com wrote:

@treymiller I need to move to a more sensible asset pipeline, or really tweak what we have going on here. It is too hard to test locally. Assets are basically images, css, js, etc. I will work on this throughout the day.

**Comment by treymiller** [comment]
Internal Server Error on the front-end right now…

Trey
Triad Tech Services
336.912.2110
triadts.com

On Apr 7, 2014, at 9:30 AM, Braden Douglass notifications@github.com wrote:

Comments created to issue from 455ddb6

 delete text from the MailChip snippet code that says "Sign up to be notified when we launch the Beta”

 turn off the My Account link in the header, since it’s broken right now

 take out some of the black horizontal space at the footer around the Twitter link, and change text to “You shouldn’t follow us on…” and ideally put the Twitter logo/link on the same horizontal line

**Comment by braidn** [comment]
@treymiller should be fixed

**Comment by treymiller** [comment]
Back up…thanks for whatever you did…

Trey
Triad Tech Services
336.912.2110
triadts.com

On Apr 7, 2014, at 10:13 AM, Braden Douglass notifications@github.com wrote:

@treymiller what does wercker say is the problem?

**Comment by treymiller** [comment]
Also, I would love it if the twitter line said “you shouldn’t follow us on” not “should”…playful, ironic, and subtly underscoring our thesis that email is enough...

Trey
Triad Tech Services
336.912.2110
triadts.com

On Apr 7, 2014, at 10:13 AM, Braden Douglass notifications@github.com wrote:

@treymiller what does wercker say is the problem?

**Comment by braidn** [comment]
@treymiller You and I are definitely inline with our humor. However, do you think that everyone will get that? Maybe we should be more in their face with the idea.

**Comment by treymiller** [comment]
I think most people won’t read it carefully enough to see it, and the ones who notice will
- think it’s weird and ignore the link and skip the click
- think it’s weird and click anyway
- think it’s somewhere between mildly ironic and hilarious, and click or not click with some range of enthusiasm

The true business purpose of our Twitter account will be for system status, so we could can the “follow me” model and say “for current system status, follow us on twitter"

Trey
Triad Tech Services
336.912.2110
triadts.com

On Apr 7, 2014, at 10:54 AM, Braden Douglass notifications@github.com wrote:

@treymiller You and I are definitely inline with our humor. However, do you think that everyone will get that? Maybe we should be more in their face with the idea.

**Comment by braidn** [comment]
Got it.

**Comment by treymiller** [comment]
For:

tos.md
why.md

Weird problem with markdown interpretation somewhere between prose.io and html…the previews look fine in prose.io, but the web page doesn’t interpret lists (UL or OL) correctly…thoughts?

Not the biggest problem right now, but I’d love to send an email out to subscribers this afternoon announcing 2nd place, pointing to the Why page (it’s the best we have for now), and encouraging them to spread the word...

Trey
Triad Tech Services
336.912.2110
triadts.com

On Apr 7, 2014, at 10:54 AM, Braden Douglass notifications@github.com wrote:

@treymiller You and I are definitely inline with our humor. However, do you think that everyone will get that? Maybe we should be more in their face with the idea.

**Comment by braidn** [comment]
Changed the + to \* . I believe the + are a GitHub flavored markdown thing. We shall see though.

**Comment by braidn** [comment]
@jordancauley can you bake in some css love for these to help out. The generated HTML is correct.

**Comment by ghost** [comment]
I'm going to rewrite the whole frontend

**Comment by treymiller** [comment]
Is there something in the CSS that’s mistreating UL and OL? That seems the most likely possibility...

Trey
Triad Tech Services
336.912.2110
triadts.com

On Apr 7, 2014, at 12:02 PM, Braden Douglass notifications@github.com wrote:

@jordancauley can you bake in some css love for these to help out. The generated HTML is correct.

**Comment by braidn** [comment]
No, it doesn't exist. I am hot fixing it since Jordan's solution isn't really what you are looking for

**Comment by treymiller** [comment]
Thanks. I really just want it to render OK for the next few days so I can send out a follow-up email to our Sign-up people…doesn’t need to be beautiful yet…

Let me know if/when you work it out…thanks.

Trey
Triad Tech Services
336.912.2110
triadts.com

On Apr 7, 2014, at 2:29 PM, Braden Douglass notifications@github.com wrote:

No, it doesn't exist. I am hot fixing it since Jordan's solution isn't really what you are looking for

**Comment by braidn** [comment]
@treymiller yeah, that's cool. I am going to space the pieces out in the markdown and it should look better. This can get us through

**Comment by braidn** [comment]
@treymiller is wercker backed up? I am changing things on the front end and the application doesn't seem to be responding. Wonder if everything is ok there.

**Comment by treymiller** [comment]
You had a failed build about an hour ago, and 2 successes since then.

You might hold off for a bit, I’m at Beans Boro and Jordan’s here too, and he may have a new skin on the front-end sooner than expected…I didn’t ask his ETA, but I’ll probably hold off on the email to the group until he’s done that…Sorry for the crossed wires...

Trey
Triad Tech Services
336.912.2110
triadts.com

On Apr 7, 2014, at 2:59 PM, Braden Douglass notifications@github.com wrote:

@treymiller is wercker backed up? I am changing things on the front end and the application doesn't seem to be responding. Wonder if everything is ok there.
